### DISCUSSION TITLE
Zigbee2mqtt+ Raspberrymatic auf HomeAssistant OS . Zigbee2mqtt fällt aus

### DISCUSSION THREAD

**Discussion by IAmWebSA** [OpenCCU/OpenCCU#2709] [🇩🇪 Fragen&Antworten]
Guten Morgen zusammen,

Ich habe ein Raspberry Pi3 B mit HomeAssistant OS und zigbee2mqtt am laufen mit dem sonoff zigbee E Stick.

Dies hatte problemlos funktioniert.

Dann habe ich das Raspberrymatic addon installiert und auf dem PI das GPIO Module installiert ( hm-mod-rpi-pcb). Dtoverlay in der config.txt Kommentare entfernt fürs UART.

Das Modul wird auch erkannt und anlernen funktionierte. Jedoch musste ich dann feststellen, dass das zigbee2mqtt was vorher lief beendet wurde.

Und wenn ich es erneut starte. Will hängt es bei 

```
2024-04-06T07:04:55.895Z zigbee-herdsman:adapter:ezsp:uart Can't send DATA frame (0,1,0): 00000004
2024-04-06T07:04:56.401Z zigbee-herdsman:adapter:ezsp:uart ->> DATA (0,1,1): 00000004
2024-04-06T07:04:56.446Z zigbee-herdsman:adapter:ezsp:uart --> [094221a85045507e]
2024-04-06T07:04:56.487Z zigbee-herdsman:adapter:ezsp:uart -?- rewaiting (1)
Zigbee2MQTT:error 2024-04-06 09:05:01: Error while starting zigbee-herdsman
Zigbee2MQTT:error 2024-04-06 09:05:02: Failed to start zigbee
Zigbee2MQTT:error 2024-04-06 09:05:02: Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html for possible solutions
2024-04-06T07:05:01.056Z zigbee-herdsman:adapter:ezsp:uart --> Error: Error: {"sequence":1} after 4000ms
Zigbee2MQTT:error 2024-04-06 09:05:02: Exiting...
```

Blockiert Raspberrymatic mein USB Zigbee?

Vielen Dank vorab

**Comment by jens-maus** [comment]
Hast du auch das CUxD CCU-Addon innerhalb von RaspberryMatic installiert bzw. nutzt das? Denn dieses schnappt sich standardmäßig leider alle tty devices, sodass andere Prozesse die nicht mehr für sich nutzen können. Wenn das so ist müsstest du dies in der CUxD Konfiguration (`cuxd.ini`) via `TTYPARAM=NONE` Eintrag unterbinden.

So vermutlich exakt das gleiche Problem wie in https://github.com/jens-maus/RaspberryMatic/discussions/2226 und https://github.com/jens-maus/RaspberryMatic/discussions/2182  beschrieben?

**Reply by IAmWebSA** [reply]
Habe kein addon im Raspberrymatic addon installiert, ist dies standardmäßig aktiv?

Evtl. Ein Pi3B Problem mit zu wenig RAM?

Habe mal ein beelink Mini S12 bestellt, dann kann ich das HomeAssistant als proxmox VM laufen lassen und Raspberrymatic nativ auf dem PI laufen lassen.

**Reply by snookerap** [reply]
> Hast du auch das CUxD CCU-Addon innerhalb von RaspberryMatic installiert bzw. nutzt das? Denn dieses schnappt sich standardmäßig leider alle tty devices, sodass andere Prozesse die nicht mehr für sich nutzen können. Wenn das so ist müsstest du dies in der CUxD Konfiguration (`cuxd.ini`) via `TTYPARAM=NONE` Eintrag unterbinden.
> 
> So vermutlich exakt das gleiche Problem wie in #2226 und #2182 beschrieben?

Hat bei mir geholfen.
Habe PiVCCU mit CUxD installiert und der Sonoff-Stick wollte nicht mit Zigbee2MQTT, mit dem Eintrag in der Config-Datei hat es funktioniert.

VG
